lib/xboxkrnl/xboxkrnl.h

```diff
@@ -1,9 +1,14 @@
+// SPDX-License-Identifier: CC0-1.0
+
+// SPDX-FileCopyrightText: 2017-2022 Stefan Schmidt
+// SPDX-FileCopyrightText: 2018-2021 Jannik Vogel
+// SPDX-FileCopyrightText: 2018 Sean Koppenhafer
+// SPDX-FileCopyrightText: 2022 Erik Abair
+
 /**
  * @file xboxkrnl.h
  * @author Stefan Schmidt
```

Comment:
Shall we remove this, as to not be confused with licensing information?
We could also rebrand this as "XboxDev Community" or something if we want to keep the tag.

I'm fine with keeping as-is though

Reply:
Hm, removing is probably the best option. Done.